dashboard/modules/node/tests/test_node.py

```diff
@@ -489,58 +489,5 @@ def check_logs():
         check_logs, (AssertionError, ), timeout_ms=10000)
 
 
-@pytest.mark.parametrize(
-    "ray_start_cluster_head", [{
-        "include_dashboard": True
-    }], indirect=True)
-def test_errors(enable_test_module, disable_aiohttp_cache,
```

Comment:
This feature is not working anymore since we don't log exceptions to log files anymore (instead we handle this from the caller).

But I found this problem; https://github.com/ray-project/ray/issues/21810